__tests__/numbers.test.js

```diff
@@ -190,19 +190,19 @@ describe('/numbers', () => {
     });
   });
 
-  describe('GET /remainder?a={number}&b={number}', () => {
-    xit('gives the remainder of dividing 18 by 5', done => {
+  describe('POST /remainder?a={number}&b={number}', () => {
```

Comment:
Was this the test that was written incorrectly?

Reply:
Yeah, I saw the thread on slack, but had already started, so I just made the changes that were provided on there.